hasjob/templates/formlayout.html.jinja2

```diff
@@ -27,4 +28,5 @@
   {%- if header_campaign %}{{ campaign_script() }}{% endif %}
   {% assets "js_tinymce" %}<script src="{{ ASSET_URL }}" type="text/javascript"></script>{% endassets %}
   {% block footerscripts %}{% endblock %}
+  {{ anon_user_script() }}
```

Comment:
This should be in `layout.html.jinja2` only. We have a problem if we’re copy pasting an entire block between templates.

Reply:
i went through the templates, and they need some reorganization. Some blocks have been redefined in several places. I can fix them, but seems a little off topic for this PR. Maybe a separate PR for it?